core/domain/collection_domain_test.py

```diff
@@ -641,9 +619,33 @@ class SchemaMigrationUnitTests(test_utils.GenericTestBase):
 tags: []
 title: A title
 """)
+    YAML_CONTENT_V3 = ("""category: A category
+language_code: en
+nodes:
+- acquired_skills:
+  - 0
+  - 1
+  exploration_id: Exp1
+  prerequisite_skills: []
+objective: ''
+schema_version: 2
+skills:
+  0:
```

Comment:
OK. Be aware though that when you translate to JS, keys of objects are strings. http://stackoverflow.com/questions/2002923/javascript-using-integer-as-key-in-associative-array

Will this cause issues?